src/components/localization/localized-link.js

```diff
@@ -15,7 +15,7 @@ import {
 } from 'common/utility'
 import { DerivStore } from 'store'
 
-const non_localized_links = ['/careers', '/careers/']
+const non_localized_links = ['/careers', '/careers/', '/offline-plugin-app-shell-fallback', '/offline-plugin-app-shell-fallback/']
```

Comment:
this is only applicable to localized link 😅 , it has to be upon building time (gatsby-node)